lib/Sources/Tabs/URLPreviewImage.swift

```diff
@@ -1,11 +1,11 @@
 import SwiftUI
 
 struct URLPreviewImage: View {
-    let url: URL
+    let url: URL?
```

Comment:
The `nil` leaking into here further indicates the deficiant model because we are now mixing `TabCellView` that consumes this view with the concept of a `NewTabCellView` that does not. But we've failed to define that extra case clearly and are falling back on a `nil` instead. It is my bad, I didn't do a great job on the first pass, but hey we didn't need it then.